opennms-pris-dist/src/main/resources/opennms-pris.service

```diff
@@ -81,6 +81,8 @@ start() {
         echo "FAILED"
         exit 5
       fi
+      # wait a second to fork before we collect a PID
+      sleep 1
```

Comment:
You can use something like ```echo "$!" > "${PID_FILE}"``` to get the PID of the last background process. It's available right after the command isplaced in the background and you can avoid the ```sleep```.